### PR TITLE
add Email field 

### PR DESCRIPTION
**Pull request by Gytjarek**
Added Email field; Also include two additional helper fields for rendering components conditionally and for rendering field sub-labels what is used in Email component

### CODE REVIEW

File: src/utils/strings.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const strings = {
     leastOneField: "At least one field must be filled out.",
     unknownError: "An unknown error occured.",
     required: "This field is required.",
+    emailsDontmatch: "Your emails do not match.",
```
Comment (robmarshall) [inline src/utils/strings.js:8]: Please change to "Emails must match" or similar. 

"Your" is not needed. These error messages are written as what should be done, as opposed to what the user has done incorrectly. More positive.

File: src/components/General/SubLabelWrapper/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  customLabel,
+  name,
+  className,
+}) => {
+  const sub_label_class =
+    subLabelPlacement == "hidden_label"
+      ? "hidden_sub_label screen-reader-text"
+      : "";
+
+  // @TODO respect FORM settings sub label placement
```
Comment (robmarshall) [inline src/components/General/SubLabelWrapper/index.js:19]: @Gytjarek This todo doesn't seem to be needed any more.

**Comment by robmarshall** [comment]
When the email component includes a confirmation input, the sub labels to not seems to connect using the "for" and "id" attributes. This currently works for an email without a confirmation input.

File: src/components/General/ConditionalWrapper.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+const ConditionalWrapper = ({ condition, wrapper, children }) =>
```
Comment (robmarshall) [inline src/components/General/ConditionalWrapper.js:3]: This will be a helpful addition. Nice one!

File: src/utils/strings.js (hunk 1 above)
Comment (Gytjarek) [reply src/utils/strings.js:8]: Don't we want to keep the original Gravity Forms error messages? I just copied them from the plugin

File: src/components/General/SubLabelWrapper/index.js (hunk 2 above)
Comment (Gytjarek) [reply src/components/General/SubLabelWrapper/index.js:19]: Kind of, it is still needed, as I'm retrieving `subLabelPlacement` from the field settings and I get `INHERIT` when it's not changed, but we don't know what INHERIT means in this case. Therefore, we need to obtain the value from the form settings (TOP or BOTTOM)

**Comment by Gytjarek** [comment]
@robmarshall I've updated my PR, `for` attr should be fixed  when confirmation field is set, also `fieldset` should be in place

**Pull request merged**